### ISSUE TITLE
snippet-bot full scan

### ISSUE THREAD

**Issue by glasnt** [terraform-google-modules/terraform-docs-samples#746]
### TL;DR

(description to come)

### Terraform Configuration

```hcl
N/A
```

### Terraform Version

```sh
N/A
```

## snippet-bot scan result
Life is too short to manually check unmatched region tags.
Here is the result:
- [ ] [cloud_sql/postgres_instance_private_ip/main.tf:85](https://github.com/terraform-google-modules/terraform-docs-samples/blob/01fdda0/cloud_sql/postgres_instance_private_ip/main.tf#L85), tag `cloud_sql_postgres_instance_private_ip_dns` doesn't have a matching start tag.
- [ ] [cloud_sql/postgres_instance_private_ip/main.tf:74](https://github.com/terraform-google-modules/terraform-docs-samples/blob/01fdda0/cloud_sql/postgres_instance_private_ip/main.tf#L74), tag ` cloud_sql_postgres_instance_private_ip_dns` doesn't have a matching end tag.

---
Report generated by [snippet-bot](https://github.com/apps/snippet-bot).
If you find problems with this result, please file an issue at:
https://github.com/googleapis/repo-automation-bots/issues.
